src/chainparams.cpp

```diff
@@ -409,6 +413,7 @@ class CRegTestParams : public CTestNetParams
         nBlockStakeModifierlV2 = std::numeric_limits<int>::max(); // max integer value (never switch on regtest)
         // Public coin spend enforcement
         nPublicZCSpends = 350;
+        nBlockV7StartHeight = nPublicZCSpends + 1;
```

Comment:
Hmm, this line makes me noise.

The accumulator checkpoint is part of the block header.

As the chain continued moving forward and using the `nAccumulatorCheckpoint` in the block hash process. if we start with v7 blocks in the past, then most likely the id will be different.

Or well, you had that in mind and that uint256 is always zero after the publicSpend activation and its inclusion/removal doesn't change the block id.

Reply:
This is regtest

Reply:
my head totally skipped the method name.. need vacations.